src/pudl/output/eia923.py

```diff
@@ -123,13 +128,17 @@ def generation_fuel_eia923(pudl_engine, freq=None,
                                                     start_date=start_date,
                                                     end_date=end_date)
 
-    first_cols = ['report_date',
-                  'plant_id_eia',
-                  'plant_id_pudl',
-                  'plant_name_eia',
-                  'utility_id_eia',
-                  'utility_id_pudl',
-                  'utility_name_eia', ]
+    first_cols = [
+        'report_date',
+        'plant_id_eia',
+        'plant_id_pudl',
+        'plant_name_eia',
+        'utility_id_eia',
+        'utility_id_pudl',
+        'utility_name_eia',
+    ]
+    if nuclear:
+        first_cols = first_cols + ['nuclear_unit_id']
```

Comment:
imma swoop back in and suggest that we add the gfn table to the CI tests.

Reply:
What do you mean "add it to the CI tests?"

Reply:
Okay I added it to the "can I pull this at all and get data" tests.